### ISSUE TITLE
Templates that need updating don't flush after migrating

### ISSUE THREAD

**Issue by ainsleyclark** [woothemes/woocommerce#35525]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

When updating page templates to the latest version, Woo doesn't automatically pick it up as being updated.

### Expected behavior

Once the template has been overridden and updated, it should automatically flush in `wc-status`

### Actual behavior

I have to manually change themes for the status page to update.

### Steps to reproduce

1. Update an outdated page template
2. View the `wc-status` page.

### WordPress Environment

```
WordPress address (URL): https://decspets.ie
Site address (URL): https://decspets.ie
WC Version: 7.0.1
REST API Version: ✔ 7.0.1
WC Blocks Version: ✔ 8.5.2
Action Scheduler Version: ✔ 3.4.0
Log Directory Writable: ✔
WP Version: 6.1
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: –
WP Cron: –
Language: en_GB
External object cache: –

### Server Environment ###

Server Info: nginx/1.18.0
PHP Version: 7.4.30
PHP Post Max Size: 8 MB
PHP Time Limit: 30
PHP Max Input Vars: 1000
cURL Version: 7.81.0
OpenSSL/3.0.2

SUHOSIN Installed: –
MySQL Version: 8.0.31-0ubuntu0.22.04.1
Max Upload Size: 8 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 7.0.1
WC Database Prefix: wp_
Total Database Size: 271.11MB
Database Data Size: 184.74MB
Database Index Size: 86.37MB
wp_woocommerce_sessions: Data: 30.06MB + Index: 1.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 2.52MB + Index: 0.45MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 12.52MB + Index: 10.03MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.05MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_actions: Data: 2.02MB + Index: 1.64MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 2.02MB + Index: 1.33MB + Engine InnoDB
wp_cli_cookie_scan_categories: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 5.52MB + Index: 8.08MB + Engine InnoDB
wp_duplicator_pro_entities: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_duplicator_pro_packages: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailchimp_carts: Data: 0.05MB + Index: 0.00MB + Engine InnoDB
wp_mailchimp_jobs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mclean_refs: Data: 1.52MB + Index: 0.00MB + Engine InnoDB
wp_mclean_scan: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_options: Data: 13.33MB + Index: 4.20MB + Engine InnoDB
wp_postmeta: Data: 58.58MB + Index: 42.14MB + Engine InnoDB
wp_posts: Data: 6.52MB + Index: 3.81MB + Engine InnoDB
wp_social_users: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_sp_email_logs: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_term_relationships: Data: 0.41MB + Index: 0.33MB + Engine InnoDB
wp_term_taxonomy: Data: 0.06MB + Index: 0.03MB + Engine InnoDB
wp_termmeta: Data: 0.25MB + Index: 0.27MB + Engine InnoDB
wp_terms: Data: 0.06MB + Index: 0.03MB + Engine InnoDB
wp_usermeta: Data: 3.52MB + Index: 4.03MB + Engine InnoDB
wp_users: Data: 0.17MB + Index: 0.16MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.06MB + Index: 0.00MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.42MB + Index: 0.39MB + Engine InnoDB
wp_wc_cvo_cache: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.08MB + Index: 0.09MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 2.52MB + Index: 1.38MB + Engine InnoDB
wp_wc_order_stats: Data: 1.52MB + Index: 0.53MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_product_attributes_lookup: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_download_directories: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.36MB + Index: 0.70MB + Engine InnoDB
wp_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wdr_order_discounts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wdr_order_item_discounts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wdr_rules: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfblockediplog: Data: 0.08MB + Index: 0.00MB + Engine InnoDB
wp_wfblocks7: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wfconfig: Data: 2.38MB + Index: 0.00MB + Engine InnoDB
wp_wfcrawlers: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wffilechanges: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wffilemods: Data: 18.55MB + Index: 0.00MB + Engine InnoDB
wp_wfhits: Data: 3.02MB + Index: 0.23MB + Engine InnoDB
wp_wfhoover: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wfissues: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wfknownfilelist: Data: 8.52MB + Index: 0.00MB + Engine InnoDB
wp_wflivetraffichuman: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wflocs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wflogins: Data: 0.47MB + Index: 0.16MB + Engine InnoDB
wp_wfls_2fa_secrets: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wfls_settings: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfnotifications: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfpendingissues: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wfreversecache: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfsnipcache: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wfstatus: Data: 0.20MB + Index: 0.14MB + Engine InnoDB
wp_wftrafficrates: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpfm_backup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpmelon_advbedit_temp: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_yoast_indexable: Data: 5.52MB + Index: 3.25MB + Engine InnoDB
wp_yoast_indexable_hierarchy: Data: 0.22MB + Index: 0.33MB + Engine InnoDB
wp_yoast_migrations: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_yoast_primary_term: Data: 0.13MB + Index: 0.14MB + Engine InnoDB
wp_yoast_prominent_words: Data: 0.20MB + Index: 0.25MB + Engine InnoDB
wp_yoast_seo_links: Data: 0.27MB + Index: 0.23MB + Engine InnoDB

### Post Type Counts ###

acf-field: 217
acf-field-group: 18
attachment: 4179
aw_workflow: 4
blocks: 5
cookielawinfo: 6
custom_css: 1
customize_changeset: 1
forms: 59
lshowcase: 7
nav_menu_item: 101
news: 24
oembed_cache: 55
page: 17
post: 7
product: 1751
product_variation: 1236
revision: 294
services: 3
shop_coupon: 180
shop_order: 6207
shop_order_refund: 148
shop_subscription: 5
team: 10
wp_global_styles: 1
wpautoterms_page: 3
wpcf7_contact_form: 1
wpl_lcp_shortcodes: 1
wpl_logo_carousel: 6

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (30) ###

Woocommerce BOIPA: by  – 1.1.0
ACF Content Analysis for Yoast SEO: by Thomas Kräftner
ViktorFroberg
marol87
pekz0r
angrycreative
Team Yoast – 3.0.1

Advanced Custom Fields PRO: by WP Engine – 6.0.3
Classic Editor: by WordPress Contributors – 1.6.2
Currency Switcher for WooCommerce: by WP Wham – 2.15.1
Facebook for WooCommerce: by Facebook – 2.6.29
Hide Categories and Products for Woocommerce: by N.O.U.S. Open Useful and Simple – 1.2.5
Jetpack: by Automattic – 11.5.1
Klarna Order Management for WooCommerce: by klarna
krokedil – 1.7.0

Klarna Payments for WooCommerce: by krokedil
klarna
automattic – 2.12.1

Klaviyo: by Klaviyo
Inc. – 3.0.4

NitroPack: by NitroPack LLC – 1.5.17
Permalink Manager for WooCommerce: by BeRocket – 1.0.7.6
Post Types Order: by Nsp Code – 1.9.9.1
SparkPost: by SparkPost – 3.2.5
Trustpilot-reviews: by Trustpilot – 2.5.914
Advance Ecommerce Tracking Premium: by theDotstore – 3.6.4
WooCommerce Advanced Bulk Edit: by George Iron – 4.5
WooCommerce Extended Coupon Features FREE: by Soft79 – 3.2.9
WooCommerce Dynamic Pricing: by Element Stark – 3.1.28
WooCommerce PayPal Checkout Gateway: by WooCommerce – 2.1.3
WooCommerce Stripe Gateway: by WooCommerce – 6.9.0
WooCommerce Google Analytics Integration: by WooCommerce – 1.5.16
WooCommerce Google Ads Dynamic Remarketing: by SweetCode – 1.8.2
WooCommerce PayPal Payments: by WooCommerce – 1.9.5
WooCommerce Raneri Free Shipping: by Raneri Web Design Snc – 1.49
WooCommerce Subscriptions: by WooCommerce – 4.6.0
WooCommerce: by Automattic – 7.0.1
Wordfence Security: by Wordfence – 7.7.1
Yoast SEO Premium: by Team Yoast – 15.2

### Inactive Plugins (2) ###

Mailchimp for WooCommerce: by Mailchimp – 2.7.6
Media Cleaner (Pro): by Jordy Meow – 6.2.8

### Dropin Plugins (1) ###

advanced-cache.php: advanced-cache.php

### Settings ###

API Enabled: ✔
Force SSL: –
Currency: EUR (€)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: ✔
Enforce Approved Product Download Directories: –

### WC Pages ###

Shop base: #13069 - /shop/
Basket: #13072 - /cart/
Checkout: #13 - /checkout/
My account: #14 - /my-account/
Terms and conditions: #13073 - /terms-conditions/

### Theme ###

Name: Dec's Pets 2022 Theme
Version: 1
Author URL: https://ainsleyclark.com
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build
personally we recommend using a child theme. See: How to create a child theme

WooCommerce Support: ✔

### Templates ###

Overrides: decspets2022/woocommerce/archive-product.php
decspets2022/woocommerce/cart/cart-empty.php
decspets2022/woocommerce/cart/cart-shipping.php
decspets2022/woocommerce/cart/cart-totals.php
decspets2022/woocommerce/cart/cart.php
decspets2022/woocommerce/cart/proceed-to-checkout-button.php
decspets2022/woocommerce/cart/shipping-calculator.php
decspets2022/woocommerce/checkout/cart-errors.php
decspets2022/woocommerce/checkout/form-billing.php
decspets2022/woocommerce/checkout/form-checkout.php
decspets2022/woocommerce/checkout/form-coupon.php
decspets2022/woocommerce/checkout/form-login.php
**decspets2022/woocommerce/checkout/form-pay.php version 5.2.0 is out of date. The core version is 7.0.1**
decspets2022/woocommerce/checkout/form-shipping.php
decspets2022/woocommerce/checkout/order-receipt.php
decspets2022/woocommerce/checkout/payment-method.php
decspets2022/woocommerce/checkout/payment.php
decspets2022/woocommerce/checkout/review-order.php
decspets2022/woocommerce/checkout/terms.php
decspets2022/woocommerce/checkout/thankyou.php
decspets2022/woocommerce/content-product-cat.php
decspets2022/woocommerce/content-product.php
decspets2022/woocommerce/content-single-product.php
decspets2022/woocommerce/content-widget-price-filter.php
decspets2022/woocommerce/emails/email-footer.php
decspets2022/woocommerce/global/form-login.php
decspets2022/woocommerce/global/quantity-input.php
decspets2022/woocommerce/global/wrapper-end.php
decspets2022/woocommerce/global/wrapper-start.php
decspets2022/woocommerce/loop/loop-end.php
decspets2022/woocommerce/loop/loop-start.php
decspets2022/woocommerce/loop/no-products-found.php
decspets2022/woocommerce/loop/orderby.php
decspets2022/woocommerce/loop/pagination.php
decspets2022/woocommerce/loop/result-count.php
decspets2022/woocommerce/myaccount/dashboard.php
decspets2022/woocommerce/myaccount/form-add-payment-method.php
decspets2022/woocommerce/myaccount/form-edit-account.php
decspets2022/woocommerce/myaccount/form-edit-address.php
decspets2022/woocommerce/myaccount/form-login.php
decspets2022/woocommerce/myaccount/form-lost-password.php
decspets2022/woocommerce/myaccount/form-reset-password.php
decspets2022/woocommerce/myaccount/my-account.php
decspets2022/woocommerce/myaccount/my-address.php
decspets2022/woocommerce/myaccount/navigation.php
decspets2022/woocommerce/myaccount/orders.php
decspets2022/woocommerce/myaccount/payment-methods.php
decspets2022/woocommerce/myaccount/view-order.php
decspets2022/woocommerce/notices/error.php
decspets2022/woocommerce/notices/notice.php
decspets2022/woocommerce/notices/success.php
decspets2022/woocommerce/order/order-again.php
decspets2022/woocommerce/order/order-details-customer.php
decspets2022/woocommerce/order/order-details.php
decspets2022/woocommerce/single-product/add-to-cart/external.php
decspets2022/woocommerce/single-product/add-to-cart/simple.php
decspets2022/woocommerce/single-product/add-to-cart/variable.php
decspets2022/woocommerce/single-product/add-to-cart/variation-add-to-cart-button.php
decspets2022/woocommerce/single-product/meta.php
decspets2022/woocommerce/single-product/price.php
decspets2022/woocommerce/single-product/product-attributes.php
decspets2022/woocommerce/single-product/product-image.php
decspets2022/woocommerce/single-product/rating.php
decspets2022/woocommerce/single-product/related.php
decspets2022/woocommerce/single-product/short-description.php
decspets2022/woocommerce/single-product/tabs/additional-information.php
decspets2022/woocommerce/single-product/tabs/description.php
decspets2022/woocommerce/single-product/tabs/tabs.php
decspets2022/woocommerce/single-product/title.php

### Subscriptions ###

WCS_DEBUG: ✔ No
Subscriptions Mode: ✔ Live
Subscriptions Live URL: https://decspets.ie
Subscriptions Template Theme Overrides: decspets2022/woocommerce/cart/cart-recurring-shipping.php
decspets2022/woocommerce/myaccount/my-subscriptions.php
decspets2022/woocommerce/myaccount/related-orders.php
decspets2022/woocommerce/myaccount/related-subscriptions.php
decspets2022/woocommerce/myaccount/subscription-details.php
decspets2022/woocommerce/myaccount/subscription-totals-table.php
decspets2022/woocommerce/myaccount/subscription-totals.php
decspets2022/woocommerce/myaccount/subscriptions.php
decspets2022/woocommerce/myaccount/view-subscription.php
decspets2022/woocommerce/single-product/add-to-cart/subscription.php
decspets2022/woocommerce/single-product/add-to-cart/variable-subscription.php

Subscription Statuses: wc-pending-cancel: 2
wc-cancelled: 3

WooCommerce Account Connected: ✔ Yes
Active Product Key: ✔ Yes
Report Cache Enabled: ✔ Yes
Cache Update Failures: ✔ 0 failure

### Store Setup ###

Country / State: Ireland — Wexford

### Subscriptions by Payment Gateway ###

Stripe: wc-cancelled: 3
wc-pending-cancel: 2

### Payment Gateway Support ###

PayPal Checkout: products
refunds
subscriptions
subscription_cancellation
subscription_reactivation
subscription_suspension
multiple_subscriptions
subscription_payment_method_change_customer
subscription_payment_method_change_admin
subscription_amount_changes
subscription_date_changes

Stripe: products
refunds
tokenization
add_payment_method
subscriptions
subscription_cancellation
subscription_suspension
subscription_reactivation
subscription_amount_changes
subscription_date_changes
subscription_payment_method_change
subscription_payment_method_change_customer
subscription_payment_method_change_admin
multiple_subscriptions

Klarna Payments: products
refunds

### Currency Switcher Settings ###

WooCommerce Currency Switcher Plugin: 
Currency switcher on per product basis: 
Revert to shop's default currency on checkout: 
Apply currency conversion to fixed amount coupons: 
Add currency selector for fixed amount coupons: 
Apply currency conversion to shipping amount: 
Apply currency conversion to minimum order amount required for free shipping: 
Apply currency conversion to cart fees: 
Apply currency conversion to WooCommerce Price Filter Widget: 
Switcher placement: 
Switcher wrapper: 
Switcher item format: 
Link list switcher separator: 
Reposition page after currency switch: 
Rounding: 
Number of decimals: 
Make "pretty price": 
Apply rounding and "pretty price" to shipping rates: 
Apply rounding and "pretty price" to shop's default currency: 
Order currency: 
Format price: 
Currencies: 
Currency (shop's default): [EUR] Euro
Total additional currencies: 2
Additional currency #1: yes
alg_currency_switcher_currency_1: EUR
Additional currency #2: yes
alg_currency_switcher_currency_2: GBP
Exchange Rates: 
Exchange rates updates: twicedaily
Exchange rates server: ecb
Free Currency Converter API Key: not set
CoinMarketCap API Key: 
Exchange rates offset: 0
#2 EUR/GBP (Pound sterling): 0.87478000
alg_currency_switcher_exchange_rate_server_EUR_GBP: default
alg_currency_switcher_exchange_rate_offset_type_EUR_GBP: default
percent: 0
Currency Countries (by IP): 
Currency Countries (by IP): yes
Enter countries as comma separated text: no
Override country: disabled
#1 [EUR] Euro: Array
(
)
#2 [GBP] Pound sterling: Array
(
)
Currency Languages (Locales): 
Currency Languages (Locales): 
Enter locales as comma separated text: 
Always use locale to assign currency: 
#1 [EUR] Euro: Array
(
    [0] => 
)
#2 [GBP] Pound sterling: Array
(
    [0] => 
)
Price Formats: 
#0 [EUR] Euro: left
Additional currency code position (optional): 
Thousand separator: ,
Decimal separator: .
Number of decimals: 2
Currency Symbol: 
#1 [EUR] Euro: 
Additional currency code position (optional): 
Thousand separator: 
Decimal separator: 
Number of decimals: 
Currency Symbol: 
#2 [GBP] Pound sterling: 
Additional currency code position (optional): 
Thousand separator: 
Decimal separator: 
Number of decimals: 
Currency Symbol: 
Flags: 
#0 [EUR] Euro: 
#2 [GBP] Pound sterling: 
Price conversion method: 
Fix mini cart: 
Disable on URI: 
Additional price filters: 
Price filters to remove: 
Session save path: 


### Klarna Countries ###

Production Countries: IE
Test Countries: No countries selected

### WooCommerce PayPal Payments ###

Onboarded: –
Shop country code: IE
WooCommerce currency supported: ✔
Advanced Card Processing available in country: –
Pay Later messaging available in country: –
Webhook status: –
Vault enabled: –
Logging enabled: –
Reference Transactions: –
Used PayPal Checkout plugin: ✔
Tracking enabled: –

### Admin ###

Enabled Features: activity-panels
analytics
coupons
customer-effort-score-tracks
experimental-products-task
experimental-import-products-task
experimental-fashion-sample-products
experimental-product-tour
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
multichannel-marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page

Disabled Features: minified-js
new-product-management-experience
settings

Daily Cron: ✔ Next scheduled: 2022-11-05 09:01:19 +00:00
Options: ✔
Notes: 69
Onboarding: completed

### Action Scheduler ###

Canceled: 26
Oldest: 2022-10-10 15:00:28 +0000
Newest: 2022-11-01 17:26:28 +0000

Complete: 4,487
Oldest: 2022-10-08 18:58:37 +0000
Newest: 2022-11-08 18:35:40 +0000

Failed: 5
Oldest: 2021-11-04 15:53:52 +0000
Newest: 2022-03-15 18:29:49 +0000

Pending: 2
Oldest: 2022-11-09 09:33:04 +0000
Newest: 2022-11-09 12:37:59 +0000


### Status report information ###

Generated at: 2022-11-08 19:18:47 +00:00
```

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by lukecav** [comment]
https://wordpress.org/support/topic/cant-dismiss-admin-note-about-template-files/

**Comment by ainsleyclark** [comment]
Just FYI this has occurred on all versions, not just 7.

**Comment by cena** [comment]
Possibly related: https://wordpress.org/support/topic/fatal-error-after-woocommerce-update-8/

**Comment by dominic-p** [comment]
Just by way of confirmation I'm seeing the same behavior after updating to 7.1.0. Only a theme change causes the updated template files to be recognized.

**Comment by samrat-biswas-git** [comment]
Hi @ainsleyclark - thanks for such a detailed report, it's really helpful.

I know this has been open for a very long time, and I appreciate your patience. We're currently going through some older WooCommerce issues, identifying what still needs fixing, and tidying up what's already fixed or out of scope.

I'd like to get an engineer's eyes on this before we make a call, so I've flagged it for developer feedback. Someone will take a proper look at the repro steps and how it's behaving now.

The bit that needs a closer look: The report may still reflect a real compatibility or cache issue, but trunk code makes the failure mode ambiguous. This needs targeted reproduction rather than a premature invalid or ready_eng call.
